ablog/post.py

```diff
@@ -675,14 +675,10 @@ def generate_atom_feeds(app):
 
     blog = Blog(app)
 
-    url = blog.blog_baseurl
-    if not url:
+    base_url = blog.blog_baseurl
+    if not base_url:
         return
 
-    base_url = url
-    if len(blog.blog_languages) > 1:
-        base_url = os_path_join(url, app.config.language)
```

Comment:
I think this had a purpose but right now I can't work out when the language would be in the URL.